### ISSUE TITLE
[Bug]: Perplexity: custom stop words are not implemented for completions.

### ISSUE THREAD

**Issue by fullstackwebdev** [BerriAI/litellm#867]
### What happened?

custom stop words are not implemented for completions.

command ran:
```sh
export PERPLEXITYAI_API_KEY=pplx-....
litellm -m perplexity/mistral-7b-instruct  --port 6000 --debug
```

is there some way to optionally remove 'stop' from being sent to Perplexity.AI?

### Relevant log output

```shell
POST Request Sent from LiteLLM:
curl -X POST \
https://api.perplexity.ai \
-d '{'model': 'mistral-7b-instruct', 'messages': [{'role': 'system', 'content': 'Context information is below.\n--------------------\n\n--------------------'}, {'role': 'user', 'content': 'You are a helpful AI assistant.\n    \nUse the the context to answer the question at the end.\n\nAlways answer in as much detail as possible.\n\nIf you don\'t know the answer, just say you don\'t know. DO NOT try to make up an answer.\n\nIf the question is not related to the context, politely respond "I am sorry, but I cannot answer that question as it is not related to what I have been trained on".  \nRespond in correct markdown format only, make a lengthy reponse.'}, {'role': 'user', 'content': 'sdfsdfsdfds'}], 'temperature': 0, 'top_p': 1.0, 'stream': True, 'stop': '<|im_end|>', 'max_tokens': 500, 'n': 1}'


RAW RESPONSE:
<generator object OpenAIChatCompletion.streaming at 0x7f3caaffc4a0>


Logging Details Post-API Call: logger_fn - None | callable(logger_fn) - False
INFO:     127.0.0.1:47836 - "POST /v1/chat/completions HTTP/1.1" 200 OK
inside generator
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/home/fullstack/anaconda3/lib/python3.10/site-packages/uvicorn/protocols/http/httptools_impl.py", line 426, in run_asgi
    result = await app(  # type: ignore[func-returns-value]

.....


    return cast(ResponseT, self.request(cast_to, opts, stream=stream, stream_cls=stream_cls))
  File "/home/fullstack/anaconda3/lib/python3.10/site-packages/openai/_base_client.py", line 834, in request
    return self._request(
  File "/home/fullstack/anaconda3/lib/python3.10/site-packages/openai/_base_client.py", line 877, in _request
    raise self._make_status_error_from_response(err.response) from None
openai.BadRequestError: Error code: 400 - {'error': {'message': 'custom stop words are not implemented for completions.', 'type': 'unsupported_parameter', 'code': 400}}
```

**Comment by ghost** [comment]
@fullstackwebdev It looks like perplexity doesn't completely map to openai. I'll fix this today. 

You should just need to do: add `--drop_params` to your cli argument for this to work. 

Will let you know once it's fixed today.

**Comment by ghost** [comment]
Curious - what're you using the proxy for? @fullstackwebdev

**Comment by ghost** [comment]
@fullstackwebdev just pushed the fix - https://github.com/BerriAI/litellm/commit/10fe16c965511d4bd97e3100bde54b3701e021be

**Comment by ghost** [comment]
will update this ticket once a fix is in prod.

**Comment by fullstackwebdev** [comment]
for future reference
https://docs.perplexity.ai/docs/feature-roadmap
![Screenshot from 2023-11-22 14-54-07](https://github.com/BerriAI/litellm/assets/10137946/27f46198-d25d-4779-8b4c-3ebd97d267da)

**Comment by ghost** [comment]
fix for this should be live with v`1.4.0`
